xlive/H2MOD/Discord/DiscordInterface.cpp

```diff
@@ -136,6 +173,21 @@ void DiscordInterface::SetPlayerCountInfo(int current, int max)
 	updateDiscordPresence();
 }
 
+void DiscordInterface::RoundTimeReset()
+{
+	int currentRoundNumber = *(int*)((BYTE*)0x300056C6);
+	DWORD roundState = *(DWORD*)(0x3000629C); //takes on different values when transitioning from round end to new round
+	
+	if (currentRoundNumber != previousRoundNumber && roundTimeCheck != 0 && updateOnNewRound != true) {
+		updateDiscordPresence();
+		updateOnNewRound = true;
+	}
+
+	if (roundState == 1 && updateOnNewRound != false) {
```

Comment:
what does roundState == 1 actually mean? actually document all the possible values for round state.